setup.cfg

```diff
@@ -90,7 +90,7 @@ known_post_core=pandas.tseries,pandas.io,pandas.plotting
 sections=FUTURE,STDLIB,THIRDPARTY,PRE_CORE,DTYPES,FIRSTPARTY,POST_CORE,LOCALFOLDER
 
 known_first_party=pandas
-known_third_party=Cython,numpy,python-dateutil,pytz,pyarrow
```

Comment:
can u remove anything u isorted

Reply:
I didn't isort anything (other than the new files, which aren't allowed to fail).

For whatever reason, my env running `isort` thought that that `pytest` was a first party module, so it grouped it with pandas.